vm/src/stdlib/sys.rs

```diff
@@ -262,6 +262,12 @@ mod sys {
         version::get_version()
     }
 
+    #[cfg(windows)]
+    #[pyattr]
+    fn winver(_vm: &VirtualMachine) -> String {
+        version::get_winver_number()
```

Comment:
```suggestion
        // Note: This is Python DLL version in CPython, but we arbitrary fill it for compatibility
        version::get_winver_number()
```
Because this is not the implementation by specification, adding a note will be helpful.

Reply:
Done